docs: dynamic create data component (#10517)
langflow-ai/langflow: docs/docs/Components/components-processing.mdx

</Tabs>

## Dynamic Create Data

The **Dynamic Create Data** component creates a [`Data`](/data-types#data) object with configurable fields.
Define the table in the **Input Configuration** field, and the component creates corresponding input or output handles in the component.

### Use the Dynamic Create Data component in a flow

The following example demonstrates how to use a **Dynamic Create Data** component to create a structured `Data` object from multiple sources.

1. Add the **Dynamic Create Data** component to your flow.

2. To define your data's fields, in the **Input Configuration** field, click <Icon name="Table" aria-hidden="true"/> **Open table**.

3. To add rows to your table, click <Icon name="Plus" aria-hidden="true"/> **Add a new row**.
   Adding a new row creates input and output handles for the **Field Type**.
   For example, if you add a field of type `Text`, `Text` input and output handles are added to the component.
   For each new row, configure the **Field Name** and **Field Type**.

    * **Field Name**: The name of the field used as both the internal key and display label.
    * **Field Type**: The type of input field to create. The type options are:

>       * Text: Accepts `text` input or connects to `Text` or `Message` outputs.

Reviewer: ```suggestion
      * Text: Accepts direct text input or accepts `Text` or `Message` output from other components.
```

Is `Text` actually an output type? I think its considered `Message` but it doesn't have as much metadata?

      * Data: Accepts `Data` input from other components.

>       * Number: Accepts numeric input or connects to `Text` or `Message` outputs.

Reviewer: ```suggestion
      * Number: Accepts direct numeric input or accepts `Text` or `Message` outputs from other components.
```

>       * Handle: Creates a component input handle that accepts `Text`, `Data`, or `Message` outputs.

Reviewer: ```suggestion
      * Handle: Accepts `Text`, `Data`, or `Message` output from other components.
```

Is this option actually called "Handle"? Is it actually an option?
